shakedown/dcos/docker.py

```diff
@@ -9,6 +9,50 @@
 from dcos import marathon
 
 
+def docker_version(host=None, component='{{.Server.Version}}'):
+    """ Return the version of Docker [Server]
+
+        :param host: host or IP of the machine Docker is running on
+        :type host: str
+	:param component: Docker component
+	:type component: str
+        :return: Docker version
+        :rtype: str
+    """
+
+    # sudo is required for non-coreOS installs
+    command = 'sudo docker version -f {}'.format(component)
+
+    if host is None:
+        err, output = shakedown.run_command_on_master(command, None, None, False)
+    else:
+        err, output = shakedown.run_command_on_host(host, command, None, None, False)
```

Comment:
Should the `err` be reported or displayed to the user? It looks like it is silently being discarded.

Reply:
Updated.